Cogs/website.py

```diff
@@ -264,6 +264,84 @@ async def addplayer_error(self, ctx: discord.context.ApplicationContext, error):
             await ctx.respond(embed = discord.Embed(title = "Unknown error. Please contact developers to check logs", color = discord.Color.red()))
             print("Add Player error: ", error)
             raise error
+###########################################################################
+    @discord.slash_command(description = "Deletes a Player", debug_guilds=[887366492730036276])
+    @commands.has_any_role('Board Member', 'Game Officer', 'Bot Technician', 'Mod', 'Faculty Advisor')
```

Comment:
Not a note for this pr, but just pointing it out that would probably be better if we made an enum list with the possible roles once we branch to new servers since a different server may have different names.